client/yarn.lock

```diff
@@ -42,7 +42,7 @@
     imurmurhash "^0.1.4"
     slide "^1.1.5"
 
-"@babel/code-frame@7.0.0", "@babel/code-frame@^7.0.0":
+"@babel/code-frame@7.0.0", "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
```

Comment:
If there are no changes to `package.json`, there should be no changes to `yarn.lock`.